_posts/2021-06-23-cockpit-247.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Cockpit 247
+author: kkoukiou
+date: 2021-06-23
+tags: cockpit linux
+slug: cockpit-247
+category: release
+summary: Cockpit with Metrics and Virtual Machines improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 247 and Cockpit Machines version 246.
+
+## Metrics: Enable Grafana client setup
+
+"Metrics settings", [introduced in Cockpit 245](https://cockpit-project.org/blog/cockpit-245.html), has a new option to share Performance Co-Pilot data over the network. Toggling "export to network" on enables a [Redis](https://grafana-pcp.readthedocs.io/en/latest/datasources/redis.html) data source with a [pmproxy service](https://linux.die.net/man/1/pmproxy) and adjusts firewall settings as needed.
+
+![metrics settings](/images/metrics-settings-2.png)
+
+If the machine uses firewalld, you can allow pmproxy in a particular zone afterwards:
+
+![metrics firewall zone selector](/images/metrics-fw-sel.png)
+
+A remote [Grafana](https://grafana.com/) instance can then read the machine's metrics using the [PCP Grafana plugin](https://grafana-pcp.readthedocs.io/). More details are available on [the PCP metrics feature page](https://cockpit-project.org/guide/latest/feature-pcp.html).
+
+## Machines: Share host files with the guest using virtiofs
+
+Cockpit Machines can now share a host directory tree with a guest VM.
```

Comment:
Drop the "tree". Not a strong opinion though, feel free to just ignore/resolve.

Reply:
Dropped.